src/Mapbender/CoreBundle/Resources/public/mapbender.element.redlining.js

```diff
@@ -1,11 +1,11 @@
 (function($){
 
-    $.widget("mapbender.mbRedlining", {
+    $.widget('mapbender.mbRedlining', {
         options: {
             target: null,
-            display_type: 'dialog',
-            auto_activate: false,
-            deactivate_on_close: true,
+            displayType: 'dialog',
+            autoActivate: false,
+            deactivateOnClose: true,
```

Comment:
Those values come from the Element configuration, stored in the DB. You can't just rename them here.  
Even if you rename them equivalently in the Element form, all existing, running applications would effectively lose their configuration for this element.